testing/scenario_app/compile_android_aot.sh

```diff
@@ -62,15 +62,19 @@ mkdir -p "$OUTDIR"
 echo "Compiling kernel..."
 
 "$HOST_TOOLS/dart" \
+  --disable-dart-dev \
```

Comment:
why does it need this flag?

Reply:
This disables some extra work dart would do to expose the developer protocol/observatory stuff. Not essential for this PR, but should make compilation faster.

Reply:
cool. Maybe a comment would be useful (so someone doesn't remove it later)

Reply:
Done